.env.example

```diff
@@ -1,6 +1,6 @@
 DOMAIN=localhost
-DOCKER_IMAGE_BACKEND=backend
-DOCKER_IMAGE_GEOSERVER=geoserver
+DOCKER_IMAGE_BACKEND=deltares/overstroomik-service
+DOCKER_IMAGE_GEOSERVER=deltares/overstroomik-geoserver
```

Comment:
@avgils If we tag the images like this, we can do `docker-compose push` and be done with it. Docker hub autobuilds are nice, but doesn't support Git LFS, so our geoserver image failed.